python/lsst/pipe/analysis/utils.py

```diff
@@ -1322,7 +1331,8 @@ def calibrateSourceCatalog(catalog, zp):
     Requires a SourceCatalog and zeropoint as input.
     """
     # Convert to constant zero point, as for the coadds
-    fluxKeys, errKeys = getFluxKeys(catalog.schema)
+    schema = getSchema(catalog)
+    fluxKeys, errKeys = getFluxKeys(schema)
```

Comment:
I think `getFluxKeys` also requires an afwtable schema; doesn't seem to work with just a list.